### PR TITLE
first draft of dock-spawn-ts integration

### PR DESCRIPTION
**Pull request by jogibear9988**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,9 +20,11 @@
     "setup"
   ],
   "dependencies": {
+    "@iobroker/adapter-core": "^1.0.3",
     "body-parser": "^1.19.0",
     "connect-flash": "^0.1.1",
     "cookie-parser": "^1.4.4",
+    "dock-spawn-ts": "^2.18.0",
```
Comment (GermanBluefox) [inline package.json]: I think this must be in devDependencies.

Comment (jogibear9988) [reply package.json]: why???
it will be needed on runtime.
or do you mean cause it's copied during build

Comment (GermanBluefox) [reply package.json]: Yes

**Comment by GermanBluefox** [comment]
image

It is funny, but till production some fixes must be done:
- Cards width (adapter, intro) are dependent on browser width and we can get this
image
- On reload the layout will be lost.
- This feature must be configurable: ON/OFF

**Comment by GermanBluefox** [comment]
May be it will find the new life in Admin5, but for now it is too raw

**Pull request closed without merging**